### PR TITLE
Add Robust Linear Regression

### PR DESCRIPTION
**Pull request by OngYL**

### CODE REVIEW

**Review by kunyaw12 — APPROVED** [review]

**Comment by hritik5102** [comment]
Great work @OngYL 🙌🏻 
Congrats your PR has been merged 🎊

File: Contributors.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Contributor:
+
+Ong Yean Lin
```
Comment (hritik5102) [inline Contributors.md:3]: There is no need of creating a contributors page. I'll mention each contributor in the readme itself.

**Review by hritik5102 — CHANGES_REQUESTED** [review]

**Comment by hritik5102** [comment]
@OngYL change the notebook path from `jupyter/Robust Linear Regression.ipynb` to `jupyter/machine_learning/Robust Linear Regression.ipynb`.  

Rename the file : `Robust Linear Regression.ipynb ` to `Robust-Linear-Regression.ipynb`

**Comment by hritik5102** [comment]
Use `git commit --amend --no-edit` to remove the conflicts of multiple commits. Read more about [here](https://www.atlassian.com/git/tutorials/rewriting-history#git-commit--amend)

**Pull request closed without merging**